openSUSE-Tumbleweed/config.kiwi

```diff
@@ -8,9 +8,10 @@
   </description>
   <preferences>
     <type image="docker">
+      <!-- This tag is not on docker hub, to avoid conflicts -->
       <containerconfig
-        name="opensuse"
-        tag="tumbleweed"
+        name="opensuse/tumbleweed"
+        tag="current"
         maintainer="SUSE Containers Team &lt;containers@suse.com&gt;"/>
```

Comment:
Why not 'latest'? I don't see the difference and is the default in Docker.

Reply:
Because `latest` will be used on the docker hub. In case there is the rpm and dockerhub available at once that might conflict. As the RPM is really only for internal use that's just some kind of "safety net".

Reply:
I see thanks